lib/stripe/util.rb

```diff
@@ -54,6 +54,7 @@ def self.object_classes
         'order' => Order,
         'order_return' => OrderReturn,
         'three_d_secure' => ThreeDSecure,
+        'apple_pay_domain' => ApplePay::Domain,
```

Comment:
Yeah I'm not fully sure, but noticed you said ApplePayDomain for the Python one. My guess is that consistency across bindings would be easier but I'm not a Ruby person so not sure if in Ruby it's cleaner your way!